st2common/st2common/models/api/rule.py

```diff
@@ -53,6 +69,49 @@ class ActionSpec(BaseAPI):
     v.update({'required': True})
 
 
+class RuleTypeAPI(BaseAPI):
+    model = RuleTypeDB
+    schema = {
+        'title': 'RuleType',
+        'description': 'A specific type of rule.',
+        'type': 'object',
+        'properties': {
+            'id': {
+                'description': 'The unique identifier for the action runner.',
+                'type': 'string',
+                'default': None
+            },
+            'name': {
```

Comment:
Just checked TriggerType. We have pack and ref too. Can we please use the same idea here? We migrated rule from name to pack + name and it was PITA. If we don't want pack, we should have a really good reason. I don't see one.